rio_tiler/io/cogeo.py

```diff
@@ -26,63 +26,37 @@
+        dataset (rasterio.io.DatasetReader or rasterio.io.DatasetWriter or rasterio.vrt.WarpedVRT, optional): Rasterio dataset. Defaults to None.
+        tms (morecantile.TileMatrixSet, optional): TileMatrixSet grid definition. Defaults to WebMercatorQuad.
+        minzoom (int, optional): Overwrite Min Zoom level. Defaults to None.
+        maxzoom (int, optional): Overwrite Max Zoom level. Defaults to None.
+        colormap (dict, optional): Overwrite internal colormap. Defaults to None.
+        nodata (int or float or str, optional): Global options, overwrite internal nodata value. Defaults to None.
+        unscale (bool, optional): Global options, apply internal scale and offset on all read operations. Defaults to None.
+        resampling_method (rasterio.enums.Resampling, optional): Global options, resampling method to use for read operations. Defaults to None.
+        vrt_options (dict, optional): Global options, WarpedVRT options to use for read operations. Defaults to None.
+        post_process (callable, optional): Global options, Function to apply after all read operations. Defaults to None.
+
+    Examples:
+        >>> with COGReader(src_path) as cog:
+            cog.tile(...)
 
-    with rasterio.open(src_path) as src_dst:
-        with WarpedVRT(src_dst, ...) as vrt_dst:
-            with COGReader(None, dataset=vrt_dst) as cog:
+        >>> # Set global options
+            with COGReader(src_path, unscale=True, nodata=0) as cog:
                 cog.tile(...)
 
-    with rasterio.open(src_path) as src_dst:
-        with COGReader(None, dataset=src_dst) as cog:
-            cog.tile(...)
+        >>> with rasterio.open(src_path) as src_dst:
+                with WarpedVRT(src_dst, ...) as vrt_dst:
+                    with COGReader(None, dataset=vrt_dst) as cog:
+                        cog.tile(...)
 
-    Attributes
-    ----------
-    filepath: str
-        Cloud Optimized GeoTIFF path.
-    dataset: rasterio.DatasetReader, optional
-        Rasterio dataset.
-
-    Properties
-    ----------
-    minzoom: int
-        COG minimum zoom level.
-    maxzoom: int
-        COG maximum zoom level.
-    bounds: tuple[float]
-        COG bounds in WGS84 crs.
-    center: tuple[float, float, int]
-        COG center + minzoom
-    colormap: dict
-        COG internal colormap.
-    info: dict
-        General information about the COG (datatype, indexes, ...)
-
-    Methods
-    -------
-    tile(0, 0, 0, indexes=(1,2,3), expression="B1/B2", tilesize=512, resampling_methods="nearest")
-        Read a map tile from the COG.
-    part((0,10,0,10), indexes=(1,2,3,), expression="B1/B20", max_size=1024)
-        Read part of the COG.
-    preview(max_size=1024)
-        Read preview of the COG.
-    point((10, 10), indexes=1)
-        Read a point value from the COG.
-    stats(pmin=5, pmax=95)
-        Get Raster statistics.
-    meta(pmin=5, pmax=95)
-        Get info + raster statistics
```

Comment:
Properties and methods are just removed?

Reply:
They should now be rendered directly by pdocs